.github/workflows/soaking.yml

```diff
@@ -225,3 +255,41 @@ jobs:
         if: always()
         run: terraform destroy -auto-approve
         working-directory: ${{ env.TERRAFORM_DIRECTORY }}
+  output-keywords:
+    if: ${{ always() }}
+    name: Output (${{ matrix.language }}, ${{ matrix.instrumentation-type }}) Layer Keyword
+    runs-on: ubuntu-20.04
+    needs:
+      - soaking-test
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ go, java, nodejs, python ]
+        instrumentation-type: [ wrapper ]
+        include:
+          - language: java
+            instrumentation-type: agent
+    steps:
+      - name: Did any of the soak tests for the layer fail?
+        id: did-any-of-the-soak-tests-fail
+        continue-on-error: true
+        run: |
+          DID_ANY_LAYER_SOAK_TEST_FAIL=$(
+            echo '${{ toJSON(needs.soaking-test.outputs) }}' |
+            jq '
+              ."${{ matrix.language }}-${{ matrix.instrumentation-type }}-error" == "FAILED"
+            ' || echo false
+          )
+          [[ $DID_ANY_LAYER_SOAK_TEST_FAIL == true ]]
```

Comment:
what is this for?

Reply:
This helps set the success/failure status for the step by using the bash `[[ ]]` test operator.

By using this, the next step can read the `outcome` result of this step and decide whether we should output a keyword (success) or do nothing (in the case of failure).